dcos_installer/exhibitor_migrate.py

```diff
@@ -127,7 +118,7 @@ def perform():
             print("Migration already complete")
             sys.exit(0)
         else:
-            assert_can_migrate(config)
+            validate_config(config)
```

Comment:
Good catch. I've pulled out this commit / code. Could switch to having "start" and goal arrays which get indexed into which would reduce the code here a lot, but this script is temporary / going away, so easier just to leave it out for now.